import-service/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "target": "es2017",
     "outDir": "lib"
   },
-  "include": ["./**/*.ts"],
+  "include": ["./**/*.ts", "../product-service/catalogBatchProcess.ts"],
```

Comment:
hm, I think this a workaround and you should fix it somehow, services should be independent, and by this, you are strongly connecting them
they can interact only with Outputs/Import in serverless config